docs/api/api.md

```diff
@@ -541,7 +590,7 @@ GET "/containers/json""
 |Name|Description|Schema|
 |---|---|---|
 |**Command**  <br>*optional*||string|
-|**Created**  <br>*optional*||integer (int64)|
+|**Created**  <br>*optional*||string|
```

Comment:
Here, we will make some change. `Create` field has a struct of string, other original type is init64.

Actually in Moby's API, consistency exists when `CreateAt` field is a type of string, but field `Create` is a type of int64. I think we should make this consistent.

In addition, from my point of view. I think int64 is better for time field in API struct. (If that, we need to update type again but worth it). If we return a string, client side cannot deal it sometimes, but if we return int64, users can deal it whatever they like.

@skoo87 WDYT?

Reply:
I agree with your idea, string is not better than int64. But this is not compatible with moby api. Later, We may not  be able to upgrade well.

Reply:
Yes, upgrading is a reality we must take into consideration. While at the very beginning, we also need to spend more energy in design. It is a tradeoff. But I think first thing we need to do is keep compatibility.